sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -652,13 +634,14 @@ TEST_F(RessedOnKeyChangeTest, Lock) {
   });
 
   transaction_.reseed_on_key_change = true;
-  transaction_.reseed_on_key_change_lock = true;
+  transaction_.force_masks = false;
```

Comment:
Nit. This is he default value already, so I would drop this line.

Reply:
Sounds good, done here and for all the other cases below.